.eslintrc

```diff
@@ -1,11 +1,10 @@
 {
-  "extends": "rackt",
+  "extends": ["rackt", "plugin:react/recommended"],
```

Comment:
It actually took me a bit to find it; looks like @pwmckenna adopted it at https://github.com/pwmckenna/eslint-config-rackt.

I like Airbnb because it's a bit more restrictive and has gone through the effort of setting all the rules I never think about, but if it's more than a half dozen or so overrides, it might be better to go with the above.

We'll see – I want to tackle really updating the ESLint config in a separate PR though.